en-US/Language.xml

```diff
@@ -986,6 +1030,56 @@ Numbers (move from N entry) -->
 				</listitem>
 
 			</varlistentry>
+<!-- Added entry for "easy, simple" -->
+			<varlistentry id="easy">
+				<term> Avoiding "Easy" </term>
```

Comment:
Again, I don't think this title suits the content, and we shouldn't be using punctuation in titles.